src/common/components/testimonial/index.js

```diff
@@ -9,7 +9,7 @@ export default function Testimonial(props) {
         <p>{ props.children }</p>
       </blockquote>
       <cite className={ style.pullQuoteCitation }>
-        <span className={ style.pullQuoteLogo }><img src={ props.logo } alt='logo'/></span> { props.citation }
+        <span className={ style.pullQuoteLogo }><img src={ props.logo } alt=''/></span> { props.citation }
```

Comment:
@matthewpaulthomas As much as the description has been removed, for a user whom is vision impaired this still doesn't explain to them what this logo is, who for, and why its here. Can we have a description please.